app/models/super_admin.rb

```diff
@@ -19,7 +19,7 @@
 #  message_signature      :text
 #  name                   :string           not null
 #  otp_backup_codes       :text
-#  otp_required_for_login :boolean          default(FALSE), not null
+#  otp_required_for_login :boolean          default(FALSE)
```

Comment:
@stonecharioteer This is not part of the PR. Please remove it. @tds-1 / @muhsin-k Please see if there is a mistake in the schema.